.github/workflows/projects.yml

```diff
@@ -0,0 +1,133 @@
+name: GitHub Projects Automations
+permissions: {}
+
+on:
+  issues:
+    types:
+      - assigned
+      - labeled
+
+  pull_request_target:
+    types:
+      - assigned
+      - labeled
+      - opened
+
+env:
+  ISSUE_URL: ${{ github.event.issue.html_url || github.event.pull_request.html_url }}
```

Comment:
The idea behind separating the `ISSUE_URL` environment variable is that if we were to add additional jobs (for automating future projects), this URL would be common across them, while the environment variables defined at the job level are job-specific.